faasmcli/faasmcli/util/wasm.py

```diff
@@ -0,0 +1,1104 @@
+import warnings
```

Comment:
As with the `policy.py` file this one is completely missing any comments to say what it does, could you add some?